src/joint.dia.element.js

```diff
@@ -127,7 +127,7 @@ joint.dia.Element = joint.dia.Cell.extend({
 
         opt = opt || {};
 
-        this.trigger('batch:start', _.extend({}, opt, { element: this, batchName: 'resize' }));
+        this.startBatch('resize', _.extend({}, opt, { element: this }));
```

Comment:
Maybe we should get rid of this `element` attribute equals `this` here.
The `joint.dia.Cell.prototype.startBatch` can add attribute `cell` equals `this` to the option object for all calls.